app/src/main/java/org/intelehealth/app/ayu/visit/vital/VitalCollectionFragment.java

```diff
@@ -1676,6 +1674,7 @@ public boolean isDataReadyForSaving() {
                 }
 
                 obsDTO.setUuid(AppConstants.NEW_UUID);
+                
 
                 try {
                     obsDAO.insertObs(obsDTO);
```

Comment:
I have seen that observation insertion code is repeatedly here @zKaveri20, can't we create the common function with common fields like set creator, encounter uuid?